locales/ast_ES/LC_MESSAGES/duckduckgo.po

```diff
@@ -4058,23 +4009,26 @@ msgid ""
 msgstr ""
 
 msgid "We don't track you."
-msgstr ""
+msgstr "Nun te rastrexamos."
 
 msgctxt "reasons-to-use-duckduckgo"
 msgid "We don't track you. Ever."
 msgstr "Nun te rastrexamos. Daveres."
 
+msgid "We don't track you."
+msgstr "Nun te rastrexamos."
```

Comment:
This one ended up in the file twice somehow. If you remove this one, the build should succeed.

Many thanks @enolp!

Reply:
I fixed it.

Thank you for warning me.